packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -824,8 +815,6 @@ export default class NotificationServicesController extends BaseController<
     try {
       this.#setIsUpdatingMetamaskNotifications(true);
 
-      await this.#performEnableProfileSyncing();
```

Comment:
Can we call signIn here? So we attempt to sign in if you haven't before?

Maybe a little method in the `#auth` module?
```ts
#auth = {
  ensureSignedIn: async () => {
      const isSignedIn = this.#auth.isSignedIn();
      if (!isSignedIn) {
        await this.#auth.signIn();
      }
  }
}
```